indexd/index/drivers/alchemy.py

```diff
@@ -509,18 +509,13 @@ def get(self, did):
         """
         with self.session as session:
             query = session.query(IndexRecord)
-            query = query.filter(IndexRecord.did == did)
+            query = query.filter(
+                or_(IndexRecord.did == did, IndexRecord.baseid == did)
+                ).order_by(IndexRecord.created_date.desc())
```

Comment:
Can you let a user flag here?
`def get(self, did, flag=True):`

`if flag:`
       `query = query.filter(or_ (...))`
`else:`
       `query = query.filter(IndexRecord.did == did)`

Reply:
This can be very useful for my services

Reply:
I don't think we should keep adding flags for different tweaks unless it's necessary. We can merge this and if the performance is not good, we can think about adding flags later